include/mgba/core/config.h

```diff
@@ -58,6 +60,10 @@ struct mCoreOptions {
 
 	bool videoSync;
 	bool audioSync;
+
+	// Extensions
+	bool hwExtensions;
+	uint16_t hwExtensionsFlags[HWEX_FLAGS_REGISTERS_COUNT];
```

Comment:
These should not be in the core opts since they're specific to one core.

Reply:
Removed them.